testnets/holesky/docker-compose.yml

```diff
@@ -4,23 +4,17 @@ services:
     container_name: bolt-sidecar-holesky
     restart: unless-stopped
     ports:
-      - "${BOLT_SIDECAR_PORT:-8017}:${BOLT_SIDECAR_PORT:-8017}" # Bolt RPC port (this should be opened on your firewall!)
+      # NOTE: to read these envs, it is necessary to provide them via `--env-file` or having them already set.
```

Comment:
What does this mean? You removed `env_file: ./bolt-sidecar.env` and now we need to pass it as a flag when running docker compose up?

If so, why? I don't get what the problem was with leaving the env file there

Reply:
Note: it is already like this -- the full command to spin up the docker compose for Holesky is
```
docker compose --env-file bolt-sidecar.env up -d
```
See https://github.com/chainbound/bolt/blob/lore%2Ffeat%2Fholesky-launch/testnets/holesky/README.md#L477-L477 (btw `docker compose up -d --env-file bolt-sidecar.env` is wrong, the argument should be placed before, I'm going to fix that).

That flag is needed because, unless you've them already set, these envs cannot be read in the docker compose file. Lastly, invoking docker compose with the `--env-file` and then providing the `env_file` property in the docker compose with the same file is completely redundant.

Reply:
Makes sense, ty!